features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml

```diff
@@ -549,6 +549,9 @@
 
         <PublishPasswordGrantLogin>false</PublishPasswordGrantLogin>
 
+        <DCRM>
```

Comment:
Better if we put some description about the property

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3113/commits/8218abc90dd0611e61183be14012b9fb7b9ceb3b